src/components/AccountList.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import { Button } from 'semantic-ui-react';
+
+import FilterLink from '../containers/FilterLink';
+
+const AccountList = ({ accounts, onRemove }) => (
+  <section>
+    <aside style={{ float: 'left' }}>
+      Show:
+      {' '}
+      <FilterLink filter="SHOW_ALL">
+        All
+      </FilterLink>
+      {', '}
+      <FilterLink filter="SHOW_ACTIVE">
+        Active
+      </FilterLink>
+      {', '}
+      <FilterLink filter="SHOW_INACTIVE">
+        Inactive
+      </FilterLink>
+    </aside>
+    <Button primary style={{ float: 'right', marginBottom: '20px' }}>
+      <NavLink
+        style={{ color: '#FFF' }}
+        to="/users/new"
+      >
+        Account koppelen
+      </NavLink>
+    </Button>
+    <table className="ui celled table">
+      <thead>
+        <tr>
+          <th>Naam</th>
+          <th>E-mailadres</th>
+          <th>Rollen</th>
+          <th>Actief</th>
+          <th />
+        </tr>
+      </thead>
+      <tbody>
+        {accounts.map(account => (
+          <tr key={account.id}>
+            <td>
+              <NavLink
+                to={`/users/${account.id}`}
```

Comment:
Heeft React Router misschien de mogelijkheid om routes te reversen? Ik bedoel iets zoals: https://docs.djangoproject.com/en/1.11/topics/http/urls/#reverse-resolution-of-urls .